_maps/configs/pirate_apostolis

```diff
@@ -0,0 +1,47 @@
+{
+	"map_name": "Starwalker-class carrier",
+	"prefix": "ISV",
+	"namelists": ["BEASTS", "WEAPONS"],
```

Comment:
Add the pirate namelist. What's the point of a pirate ship that doesn't use the funny pirate names?

Reply:
i couldnt find a pirate prefix... is it just called "pirate" ?

Reply:
Namelist, not prefix. It’s the same one the Libertatia uses.